README.rst

```diff
@@ -249,28 +254,56 @@ Similarly, you can query events like this:
 Saved Queries
 '''''''''''''
 
-You can manage your saved queries from the Keen python client.
+You can manage your `saved queries <https://keen.io/docs/api/?shell#saved-queries>`_ from the Keen python client.
 
 .. code-block:: python
 
+    # Create your KeenClient
+    from keen.client import KeenClient
+
+    client = KeenClient(
+        project_id="xxxx",  # your project ID
+        read_key="zzzz",
+        master_key="abcd" # Most Saved Query functionality requires master_key
+    )
+
     # Create a saved query
-    keen.saved_queries.create("name", saved_query_attributes)
+    saved_query_attributes = {
+        # NOTE : For now, refresh_rate must explicitly be set to 0 unless you
+        # intend to create a Cached Query.
+        "refresh_rate": 0,
+        "query": {
+            "analysis_type": "count",
+            "event_collection": "purchases",
+            "timeframe": "this_2_weeks",
+            "filters": [{
+                "property_name": "price",
+                "operator": "gte",
+                "property_value": 1.00
+            }]
+        }
+    }
+    client.saved_queries.create("name", saved_query_attributes)
 
     # Get all saved queries
-    keen.saved_queries.all()
+    client.saved_queries.all()
 
     # Get one saved query
-    keen.saved_queries.get("saved-query-slug")
+    client.saved_queries.get("saved-query-slug")
```

Comment:
I think it's a Django-y or Wordpress-y term for a shortened name for a resource in kebab-casing. Here it would just as well be "min-purchases-in-April". It does have to adhere to the rules laid out in the [docs](https://keen.io/docs/api/?shell#creating-a-saved-query) so basically alphanumerics, underscore and dash.